circle.yml

```diff
@@ -0,0 +1,42 @@
+machine:
+  xcode:
+    version: 9.0
+  environment:
+    XCODE_WORKSPACE: "AeroGearPush.xcworkspace"
+    XCODE_SCHEME: "AeroGearPush"
+    XCODE_SCHEME_TEST: "AeroGearPushTests"
+    SIMULATOR: "iPhone 7"
+    OS: "11.0"
+
+general:
+  branches:
+    ignore:
+      - /FH-v.*/ # Ignore release branches
+
+test:
+  override:
+    - set -o pipefail &&
+      xcodebuild
+        CODE_SIGNING_REQUIRED=NO
+        CODE_SIGN_IDENTITY=
+        PROVISIONING_PROFILE=
+        -sdk iphonesimulator
+        -destination "platform=iOS Simulator,name=${SIMULATOR},OS=${OS}"
+        -workspace $XCODE_WORKSPACE
+        -scheme $XCODE_SCHEME
+        clean build |
+      tee $CIRCLE_ARTIFACTS/xcode_raw.log |
+      xcpretty --color --report junit --output $CIRCLE_TEST_REPORTS/xcode/results.xml
+
+    - set -o pipefail &&
+      xcodebuild
+        CODE_SIGNING_REQUIRED=NO
+        CODE_SIGN_IDENTITY=
+        PROVISIONING_PROFILE=
+        -sdk iphonesimulator
+        -destination "platform=iOS Simulator,name=${SIMULATOR},OS=${OS}"
+        -workspace $XCODE_WORKSPACE
+        -scheme $XCODE_SCHEME_TEST
+        clean build test |
```

Comment:
This test scheme is not configured for running, remove the `clean build` part to only run the tests, or edit the scheme to allow it to run

Reply:
Sorry! Copy and paste detected :bowtie: